From 3242297932bfd856ccb492cd76a8e3482471c975 Mon Sep 17 00:00:00 2001
From: Johannes Lorenz <j.git@lorenz-ho.me>
Date: Tue, 22 Dec 2020 21:21:17 +0100
Subject: [PATCH 1755/1936] guimain: Add additional help comments

Additional comments are added to explain that `--embed` runs zyn as a
plugin, and that window embedding only works with NTK.
---
 src/UI/guimain.cpp | 11 ++++++-----
 1 file changed, 6 insertions(+), 5 deletions(-)

diff --git a/src/UI/guimain.cpp b/src/UI/guimain.cpp
index f23aac95..22422231 100644
--- a/src/UI/guimain.cpp
+++ b/src/UI/guimain.cpp
@@ -192,7 +192,7 @@ ui_handle_t GUI::createUi(Fl_Osc_Interface *osc, void *exit)
     {
         if (long long winId = atoll(embedId))
         {
-            // running as plugin
+            // embedId passed means running as plugin
             isPlugin = true;
             MasterUI::menu_mastermenu[11].hide(); // file -> nio settings
             MasterUI::menu_mastermenu[26].deactivate(); // misc -> switch interface mode
@@ -576,9 +576,10 @@ void watch_lo(void)
 
 const char *help_message =
 "zynaddsubfx-ext-gui [options] uri - Connect to remote ZynAddSubFX\n"
-"    --help   print this help message\n"
-"    --no-uri run without a remote ZynAddSubFX\n"
-"    --embed  window ID [Internal Flag For Embedding Windows]\n"
+"    --help               print this help message\n"
+"    --no-uri             run without a remote ZynAddSubFX\n"
+"    --embed <window ID>  internal flag to let UI run in plugin mode\n"
+"                         NTK only: Embed master window into passed window\n"
 "\n"
 "    example: zynaddsubfx-ext-gui osc.udp://localhost:1234/\n"
 "      This will connect to a running zynaddsubfx instance on the same\n"
@@ -636,4 +637,4 @@ int main(int argc, char *argv[])
     lo_watch.join();
     return 0;
 }
-#endif
+#endif
\ No newline at end of file
-- 
2.47.0

